dist/leaflet.css

```diff
@@ -557,6 +612,30 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border: 1px solid #999;
 	}
 
+/* Dark Mode */
+.leaflet-darkmode .leaflet-popup-content-wrapper,
+.leaflet-darkmode .leaflet-popup-tip {
+	background: #333;
+	color: #fff;
+	box-shadow: 0 3px 14px rgba(0,0,0,0.4);
+	}
+.leaflet-darkmode.leaflet-container a.leaflet-popup-close-button{
+	color: #ddd;

```

Comment:
Unless needed for specificity, I suggest leaving out `.leaflet-container` in `.leaflet-darkmode.leaflet-container [...]`.